app/uploaders/avatar_uploader.rb

```diff
@@ -5,4 +5,8 @@ class AvatarUploader < CarrierWave::Uploader::Base
   def extension_white_list
     %w(jpg jpeg gif png)
   end
+
+  def default_url(*args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `default_url(*)` if you want the method to accept any arguments but don't care about them.